test/gui/shared/steps/steps.py

```diff
@@ -427,9 +427,9 @@ def step(context, type, resource):
     waitForFileOrFolderToSync(context, resource, type)
 
 
-@Given(r'user has waited for (file|folder) to be synced', regexp=True)
-def step(context, type, resource):
-    waitForFileOrFolderToSync(context, resource, type)
+@Given('the user has waited for files to be synced')
+def step(context):
+    waitForRootFolderToSync(context)
```

Comment:
why remove the folder option? Maybe we need folder to sync something?

Reply:
```
@Given(r'user has waited for (file|folder) to be synced', regexp=True)
def step(context, type, resource):
    waitForFileOrFolderToSync(context, resource, type)
```

```
@Given('the user has waited for files to be synced')
def step(context):
    waitForRootFolderToSync(context)
```

The previous step was an unused step, so I refactored it and changed it into the later one. It aims to check the sync status of the root sync path. It doesn't matter whether we write `(file|folder)` or not after all it looks for the sync status of the root sync path`clientRootSyncPath`.

Reply:
The previous step was an unused step, so I refactored it and changed it into the later one. It aims to check the sync status of the root sync path. It doesn't matter whether we write `(file|folder)` or not after all it looks for the sync status of the root sync path`clientRootSyncPath`.